src/app/wallets/ln-send-payment.ts

```diff
@@ -483,7 +493,16 @@ const executePaymentViaLn = async ({
         })
     if (payResult instanceof LnPaymentPendingError) return PaymentSendStatus.Pending
 
-    const settled = await ledgerService.settlePendingLnPayments(paymentHash)
+    const payment = await lndService.lookupPayment({ pubkey, paymentHash })
+    if (payment instanceof Error) return payment
```

Comment:
do we always go through this route if the payment had already failed?

Reply:
As it is now yes. The intention is that if the payment fails, we can still get back a `status: "failed"` object from lnd with useful metadata.